### PR TITLE
added ignore file

### PR DESCRIPTION
**Pull request by GiantRobato**
default ignore for vue

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.DS_Store
+Thumbs.db
+db.json
```
Comment (ericpak) [inline .gitignore:3]: @lottiely This is where you comment

Comment (GiantRobato) [reply .gitignore:3]: merrrrrh

Comment (Wakahisa) [reply .gitignore:3]: Hi, I am commenting, too.

**Review by lottiely — APPROVED** [review]
This is my first time approving. 👍

**Pull request merged**